### DISCUSSION TITLE
Recommended colorscheme

### DISCUSSION THREAD

**Discussion by aareman** [nvim-orgmode/orgmode#226] [Q&A]
Any recommended colorschemes for this plugin?

**Comment by jgollenz** [comment]
I am currently using [gruvbox.nvim](https://github.com/ellisonleao/gruvbox.nvim), but I can't really recommend it. The agenda (at least in dark-mode) only uses different shades of red.

![shades-of-red](https://user-images.githubusercontent.com/12050098/156642650-d34e60e6-f6b2-4228-98b3-33b963eac164.png)


Before that I used [gruvbox-flat.nvim](https://github.com/eddyekofo94/gruvbox-flat.nvim) which I recall to work better. But I didn't like the low contrast of the theme itself. Since I don't spend most of my time looking at my org agenda it's not that big of a deal right now, but I want to get it fixed at one point

**Comment by kristijanhusak** [comment]
https://github.com/RRethy/nvim-base16 worked fine for me for some time, and now I'm using https://github.com/rmehri01/onenord.nvim which also works great.

**Comment by refaelsh** [comment]
I use this one: https://github.com/Mofiqul/dracula.nvim. It's not that much great for this plugin, but much better than gruvbox.

P.S.
I simply love dracula too much :-)
